crates/ruff_db/src/source.rs

```diff
@@ -136,6 +137,16 @@ enum SourceTextKind {
     Notebook(Box<Notebook>),
 }
 
+impl get_size2::GetSize for SourceTextKind {
+    fn get_heap_size(&self) -> usize {
+        match self {
+            SourceTextKind::Text(text) => text.get_heap_size(),
+            // TODO: The `get-size` derive does not support ignoring enum variants.
+            SourceTextKind::Notebook(_) => 0,
```

Comment:
What's the reason for ignoring the notebook variant here?

Reply:
Fair enough. It could make sense extending the TODO comment to why it's currently excluded.